SUMMARY.md

```diff
@@ -150,6 +150,7 @@
   * ZGW
     * [Documents](/using-valtimo/zgw/documents/zgw-documents.md)
 * [Extending Valtimo](extending-valtimo/extending-valtimo.md)
+  * [Branching and release strategy](extending-valtimo/branching-and-release-strategy.md)
```

Comment:
I don't think this is the right location. "Extending Valtimo" is about how you can use certain valtimo features that require code in your implementation to use. This is not about implementing valtimo, but about developing valtimo itself. I would suggest putting it on the same level as the contributing to valtimo page. Or maybe as a sub page of that.